evaluation/src/main/java/com/formulasearchengine/mlp/evaluation/EvaluateCommand.java

```diff
@@ -16,19 +16,28 @@
         "\n1,matched,\"W\",\"van der waerden number\"")
   private String in = "";
 
-  public String getGold() {
-    return gold;
-  }
 
   @Parameter(names = {"-gold"},
     description = "The gold standard file")
   private String gold = "";
 
+  @Parameter(names = {"-titleKey"},
+    description = "Use the title instead of the qId for the matching")
```

Comment:
The title and the qId are synonymous as long as there is only one formula per title, which is the case. Also as far as I understood the relation between the gold standard and the eval_dataset the matching between these two is done by the title anyway. Since currently it is necessary to parse the gold standard anyway to get the fId we can remove this again.